features/step_definitions/publisher_steps.rb

```diff
@@ -0,0 +1,113 @@
+When "I go to add an artefact" do
+  click_on "Add artefact"
+  expect(page).to have_content("New artefact")
+end
+
+When "I create a draft place" do
+  fill_in "Title", with: publication_title
+  fill_in "Slug", with: parameterize(publication_title)
+  select "Place", from: "Format"
+  click_button "Save and go to item"
+end
+
+When "I preview the draft place" do
+  click_on "Preview"
+
+  wait_until do
+    visit page.current_url
+    page.has_no_text?(:all, /not found/i)
```

Comment:
Can you use the status code of `200` instead? I think this'll be less likely to change than the text.